include/GUI.h

```diff
@@ -27,12 +27,13 @@ enum GUIState { MAIN_MENU, PAUSE_MENU, HUD, SETTINGS, CONTROL_SETTINGS, TERMINAT
 class GUI
 {
 public:
-    GUI(ConfigData &data, gui::IGUIEnvironment &guiEnvironment);
+    GUI(ConfigData &data, gui::IGUIEnvironment &guiEnvironment, video::IVideoDriver *driver);
```

Comment:
It's much better to use a reference instead of a pointer to video::IVideoDriver here